librz/cons/dietline.c

```diff
@@ -1888,6 +1890,7 @@ RZ_API const char *rz_line_readline_cb(RzLineReadCallback cb, void *user) {
 						break;
 					case 0x37: // HOME xrvt-unicode
 						rz_cons_readchar();
+						break;
```

Comment:
same here... mmm need to test all these things.

Reply:
Added a fall-through comment to indicate that.